src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -201,36 +201,26 @@ def build_sdfg_from_itir(
         *args:		        Arguments for which the fencil should be called.
         offset_provider:	The set of offset providers that should be used.
         auto_optimize:	    Apply DaCe's `auto_optimize` heuristic.
-        for_gpu:		    Performs the translation for GPU, defaults to `None`.
+        on_gpu:		        Performs the translation for GPU, defaults to `False`.
         column_axis:		The column axis to be used, defaults to `None`.
         lift_mode:		    Which lift mode should be used, defaults `FORCE_INLINE`.
         **kwargs:           Except `run_on_gpu` all are ignored.
```

Comment:
This doc line should be updated. Can we remove `kwargs` from this function?

Reply:
The docstring definitely has to be updated.

It is possible to remove `**kwargs` from function, but this has some implications, especially at where we call the function, since we have to explicitly filter out arguments the `build_sdfg_from_itir()` (see 014e1bae).
From an aesthetic pov I would revert that commit, but I am fine if we keep the change, since it is more explicit.